src/main/antlr4/au/csiro/data61/aap/elf/parsing/Ethql.g4

```diff
@@ -6,7 +6,20 @@
 grammar Ethql;
 
 document
-    : statement* EOF
+    : (connection outputFolder)? statement* EOF
```

Comment:
Consider also the possibility of "outputFolder" and then "connection". The order of the keywords should not matter, and therefore increases usability.